aiida/cmdline/commands/cmd_export.py

```diff
@@ -189,11 +193,11 @@ def migrate(input_file, output_file, force, silent, archive_format):
 
         new_version = verify_metadata_version(metadata)
 
-        with open(folder.get_abs_path('data.json'), 'w') as handle:
-            json.dump(data, handle)
+        with io.open(folder.get_abs_path('data.json'), 'w', encoding='utf8') as fhandle:
+            fhandle.write(six.text_type(json.dumps(data, ensure_ascii=False)))
```

Comment:
I am really not convinced that this is a portable and maintainable solution, see #2142

Reply:
Now resolved.